### PR TITLE
Docs/reviewer add footer

### PR DESCRIPTION
**Pull request by kometenstaub**
## Edited

- checklist for reviewers
- scripting MOC

## Added

- "Adding footers" note

## Checklist
- [x] before creating a new note, I searched the vault
- [x] new notes have the `.md` extension
- [ ] (if applicable) attached images have descriptive file names
- [ ] (if applicable) for new notes in the folder "04 - Guides, Workflows, & Courses", I added a link to them in one of the "for {group X}" overviews: https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses

### CODE REVIEW

**Comment by claremacrae** [comment]
> It might be a good idea to add a notice of the GitHub action where I added the reference to the `add_footer` script. It still has a reference to the MOC script, and I didn't check the `update_hub` action..
> 
> I'll keep this PR open until I open an issue about it so that it's tracked.

Yeah. It's all got quite complicated... It was so much easier to update it when it was in the Wiki, but I lost that vote...

Perhaps we could pair on all these scripts at some point - as I've tried several times to update them, and got stuck every time, especially on the `update_releases.py` one.

File: 00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.01 Structure/Checklist for reviewing Pull Requests.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ When reviewing:
 After merging:
 
 - Do the MOCs need updating? See [[Updating MOC files]]
+- Add footers if a new note has been added. See [[Adding footers]]
```
[the remark this answers is not in the document]
Comment (kometenstaub) [reply 00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.01 Structure/Checklist for reviewing Pull Requests.md]: I've reworked that in this commit: [faf87bc](https://github.com/obsidian-community/obsidian-hub/pull/319/commits/faf87bcf03bfe4db4672930b7962936c4e56ac88)

File: 00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.03 Scripts and Automation/Adding footers.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## Running the script
+
+To run it:
+
+```bash
+cd obsidian-hub/.github/scripts
+python3 ./add_footer.py
+```
+
+And then review the changes and commit them.
```
[the remark this answers is not in the document]
Comment (kometenstaub) [reply 00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.03 Scripts and Automation/Adding footers.md:21]: See [a1aa46d](https://github.com/obsidian-community/obsidian-hub/pull/319/commits/a1aa46d1e51156c14b1a3037a43b9479674f9792)

File: 00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.03 Scripts and Automation/update_hub GitHub Action.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+tags:
+- seedling
+publish: true
+---
+
+# update_hub GitHub Action
+
+Links to Action:
+
+- Source of the workflow: [`update_hub.yml`](https://github.com/obsidian-community/obsidian-hub/blob/main/.github/workflows/update_hub.yml)
```
Comment (claremacrae) [inline 00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.03 Scripts and Automation/update_hub GitHub Action.md:13]: I added a link to the source of the workflow, as not everyone will know how to obtain that from looking at the logs... (I only learned that link recently...)

File: 00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.03 Scripts and Automation/update_hub GitHub Action.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- Logs from runs of workflow: [`update_hub`](https://github.com/obsidian-community/obsidian-hub/actions/workflows/update_hub.yml)
+
+Running this action will automatically create a Pull Request.
+
+To see the associated Pull Request created from a successful run, expanding the run's log and click on the 'Show Pull Request' step.
```
Comment (claremacrae) [inline 00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.03 Scripts and Automation/update_hub GitHub Action.md:18]: I added this note on how to see the associated PR...

**Review by claremacrae — APPROVED** [review]
Brilliant - thank you.

**Comment by claremacrae** [comment]
@kometenstaub FYI when #307 is merged, the review checklist item about updating the directory tree will no longer be needed as it will be done as part of `update_hub`...

I'll delete that instruction in a minute - when I've finished merging that PR...

**Comment by claremacrae** [comment]
> @kometenstaub FYI when #307 is merged, the review checklist item about updating the directory tree will no longer be needed as it will be done as part of `update_hub`...
> 
> I'll delete that instruction in a minute - when I've finished merging that PR...

Done - also updated the docs for update_hub to note that it updates the directory tree...

Over to you @kometenstaub !

**Comment by kometenstaub** [comment]
Thank you for your review and the additions. :)

**Pull request merged**